android/app/src/main/res/layout/activity_main.xml

```diff
@@ -8,18 +8,11 @@
     android:fitsSystemWindows="true"
     tools:openDrawer="start">
 
-    <android.support.design.widget.CoordinatorLayout
+    <FrameLayout
```

Comment:
@Shailesh351 
i will change that. but for tablet layout, the root layout has to be coordinator layout or drawer layout. as i dont want a drawer layout for the tablet view in that case i used a coordiantor layout. so this would mean if i cahnge the coordinator layout, i will have a coordinator layout as a root element having a child one also. is it fine? @iamareebjamal @Shailesh351